its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -165,14 +165,18 @@ public static void deleteVirtualDrive(String drive) {
     assertThat(cleanupStatus).isZero();
   }
 
-  public static Path projectDir(Path temp, String projectName) throws IOException {
-    File projectToCopy = Paths.get("projects").resolve(projectName).toFile();
-    File destination = new File(temp.toFile(), projectName).getCanonicalFile();
-    FileUtils.deleteDirectory(destination);
-    Path newFolder = Files.createDirectories(destination.toPath());
-    FileUtils.copyDirectory(projectToCopy, newFolder.toFile());
-    Files.copy(Paths.get("..", "NuGet.Config"), newFolder.resolve("NuGet.Config"));
-    return newFolder;
+  public static Path projectDir(Path temp, String projectName) {
+    try {
+      File projectToCopy = Paths.get("projects").resolve(projectName).toFile();
+      File destination = new File(temp.toFile(), projectName).getCanonicalFile();
+      FileUtils.deleteDirectory(destination);
+      Path newFolder = Files.createDirectories(destination.toPath());
+      FileUtils.copyDirectory(projectToCopy, newFolder.toFile());
+      Files.copy(Paths.get("..", "NuGet.Config"), newFolder.resolve("NuGet.Config"));
+      return newFolder;
+    } catch (IOException ex) {
+      throw new RuntimeException(ex.getMessage(), ex);
```

Comment:
That's nasty. You made the exception unchecked, right?

Reply:
Yes, exatly. Because the only thing we do with it is to propagate it to the test method and we never really care, or do something specific with it.

I wanted to avoid polluting the test codebase with pointless `throws Exception` everywhere.